docs/how_to_guides/creating_and_editing_expectations/how_to_create_expectations_that_span_multiple_batches_using_evaluation_parameters.rst

```diff
@@ -0,0 +1,139 @@
+
+
+Data Assets within a Validation operation. This is done
+
+.. admonition:: Prerequisites: This how-to guide assumes you have already:
+
+  - :ref:`Set up a working deployment of Great Expectations <getting_started>`
+  - Configured a :ref:`Datasource <Datasources>` (or Datasources) with at least two Data Assets.
+  - Also created :ref:`Expectation Suites` for those Data Assets.
+  - Have a working :ref:`Evaluation Parameter Store`. (The default in-memory store from ``great_expectations init`` can work for this.)
+  - Have a working :ref:`Validation Operator`. (The default Validation Operator from ``great_expectations init`` can work for this.)
+
+Steps
+-----
+
+In a notebook, 
+
+#. **Import great_expectations and instantiate your Data Context**
+
+    .. code-block:: python
+
+        import great_expectations as ge
+        context = ge.DataContext()
+
+#. **Instantiate two Batches**
+
+    We'll call one of these Batches the *upstream* Batch and the other the *downstream* Batch. Evaluation Parameters will allow us to use Validation Results from the upstream Batch as parameters passed into Expectations on the downstream.
+    
+    It's common (but not required) for both Batches to come from the same :ref:`Datasource <Datasources>` and :ref:`BatchKwargsGenerator`.
+
+    .. code-block:: python
+
+        upstream_batch = context.get_batch(
+            context.build_batch_kwargs("my_datasource", "my_generator_name", "my_data_asset_name_1"),
+            expectation_suite_name='my_expectation_suite_1'
+        )
+
+        downstream_batch = context.get_batch(
+            context.build_batch_kwargs("my_datasource", "my_generator_name", "my_data_asset_name_2"),
+            expectation_suite_name='my_expectation_suite_2'
+        )
+
+#. **Define an Expectation using an Evaluation Parameter on the downstream Batch.**
+
+    .. code-block:: python
+
+        eval_param_urn = 'urn:great_expectations:validations:my_expectation_suite_1:expect_table_row_count_to_be_between.result.observed_value'
+        downstream_batch.expect_table_row_count_to_equal(
+            value={
+                '$PARAMETER': eval_param_urn, # this is the actual parameter we're going to use in the validation
+                '$PARAMETER.' + eval_param_urn: 10  # this is a *temporary* value so we can execute the notebook
+            }
+        )
+    
+    The core of this is a ``$PARAMETER : URN`` pair. When Great Expectations encounters a ``$PARAMETER`` flag during validation, it will replace the ``URN`` with a value retrieved from an :ref:`Evaluation Parameter Store` or :ref:`Metrics Store`.
+    
+    This declaration above includes two ``$PARAMETERS``. The first is the real parameter that will be used after the Expectation Suite is stored and deployed in a Validation Operator. The second parameter supports immediate evaluation in the notebook.
```

Comment:
```suggestion
```

It might be worth making a "footnote" for an alternate way of doing this? Or a new how-to guide: "How to provide a temporary Evaluation Parameters when defining an expectation"

Reply:
It feels like a footnote to me, since it's not the preferred pathway. I'm reluctant to spend a lot of time documenting the nuances of this workflow, since I think it's so clunky that it's going to need to change dramatically.

Let's put it in the Additional Notes section.

Reply:
👍